plugins/core/tracing.go

```diff
@@ -47,6 +47,11 @@ func (t *Tracer) CreateEntrySpan(operationName string, extractor interface{}, op
 	defer func() {
 		saveSpanToActiveIfNotError(ctx, s, err)
 	}()
+	// if parent span is entry span, then use parent span as result
+	if tracingSpan != nil && tracingSpan.IsEntry() {
+		tracingSpan.SetOperationName(operationName)
+		return tracingSpan, nil
+	}
```

Comment:
Do we have exit span overlap mechanism? In Java agent, the entry span is always using the newest created, but exit span usually uses the first time set.

Reply:
For now, I haven't added the exit span overlap mechanism, I can add it in other PR.